src/Arcus.Messaging.Tests.Integration/ServiceBus/ServiceBusClient.cs

```diff
@@ -33,62 +35,108 @@ public ServiceBusClient(KeyRotationConfig configuration, ITestOutputHelper logge
             _logger = logger;
         }
 
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="entity"></param>
+        /// <returns></returns>
+        public async Task<AccessKeys> GetConnectionStringKeysAsync(ServiceBusEntity entity)
+        {
+            using IServiceBusManagementClient client = await CreateServiceManagementClientAsync();
+
+            return entity switch
```

Comment:
#TIL but I'm not a fan of it but it's ok.

They are going too much to an in-line approach

Reply:
Ah, yes, forgot to mention it. Wasn't sure if this is good to have in the Arcus projects bc it's 'more' advanced sometimes.
Also, they are more.growing more and more to a functional approach, that's what this 'inline' approach is what you're talking about. So naturally I'm a fan 😅, but maybe I should change this for beginners.

Reply:
It's not about beginner or non-beginner; I just don't like it from a readability perspective because if you ask me this has no function touch to it but as I said - It's ok :)

Reply:
Yes I understand, I'll change it anyway for the sake of consistency. I was referring to that it is a functional version of `switch` bc it makes the operation work as a function, with input and output, this is in line with the mindset of functional programming that _everything_ is a function. This is all by default the case in F# and it's nice to see some of these concepts in other languages. But again, we'll change it for the sake of consistency.
Thanks.for noticing.